### PR TITLE
Docs: docs to cover x-ot-span-context header

### PR DESCRIPTION
**Pull request by RomanDzhabarov**

### CODE REVIEW

File: docs/configuration/http_conn_man/headers.rst [hunk 1]
```diff
@@ -162,3 +162,15 @@ following features are available:
...
   :ref:`config_http_conn_man_headers_x-envoy-force-trace` and
   :ref:`config_http_conn_man_headers_x-client-trace-id` headers.
+
+.. _config_http_conn_man_headers_x-ot-span-context:
+
+x-ot-span-context
+_________________
+
+The *x-ot-span-context* HTTP header is used by Envoy to establish proper parent-child relationships
+between tracing spans. Envoy relies on data from *x-ot-span-context* header to extract the parent
```
Comment (junr03) [inline docs/configuration/http_conn_man/headers.rst]: data from *the* `x-ot-span-context`

File: docs/configuration/http_conn_man/headers.rst [hunk 2]
```diff
@@ -162,3 +162,15 @@ following features are available:
...
+context for the current span. For example, an egress span is a child of ingress span (if there was an ingress call).
+Envoy injects the *x-ot-span-context* header on ingress call and passes it to the local service and
```
Comment (junr03) [inline docs/configuration/http_conn_man/headers.rst]: I think this sentence is too long. I would say:

Envoy injects the `x-ot-span-context` header on ingress requests and forwards it in the request to the local service. Envoy relies on the application to propagate `x-ot-span-context` on the egress call to an upstream.

File: docs/configuration/http_conn_man/headers.rst [hunk 3]
```diff
@@ -169,8 +169,8 @@ x-ot-span-context
...
 
 The *x-ot-span-context* HTTP header is used by Envoy to establish proper parent-child relationships
-between tracing spans. Envoy relies on data from *x-ot-span-context* header to extract the parent
-context for the current span. For example, an egress span is a child of ingress span (if there was an ingress call).
-Envoy injects the *x-ot-span-context* header on ingress call and passes it to the local service and
-then relies on the application to propagate *x-ot-span-context* on the egress call to an upstream.
-See more on tracing :ref:`here <arch_overview_tracing>`.
+between tracing spans. Envoy relies on data from the *x-ot-span-context* header to extract the parent
+context for the current span. For example, an egress span is a child of an ingress
+span (if there was the ingress span). Envoy injects the *x-ot-span-context* header on ingress requests and
```
Comment (junr03) [inline docs/configuration/http_conn_man/headers.rst]: if there was *an* ingress span

Comment (RomanDzhabarov) [inline docs/configuration/http_conn_man/headers.rst]: should be "the" as I've already mentioned that span in "child of an ingress span"

Comment (junr03) [inline docs/configuration/http_conn_man/headers.rst]: Sure, but *if there was ...* is not correct with *the*. You could say (if the ingress span was present). I think the likely explanation for the phrase not working with the, is that the phrase expresses ambiguity of existence even though the object is explicit in the rest of the sentence.

Comment (junr03) [inline docs/configuration/http_conn_man/headers.rst]: But I am not a linguist, so my favored explanation is that it doesn't sound right with the

Comment (RomanDzhabarov) [inline docs/configuration/http_conn_man/headers.rst]: i'll redo to avoid confusion :)

File: docs/configuration/http_conn_man/headers.rst [hunk 4]
```diff
@@ -162,3 +162,15 @@ following features are available:
   <config_http_conn_man_runtime_random_sampling>` runtime setting or via forced tracing using the
   :ref:`config_http_conn_man_headers_x-envoy-force-trace` and
   :ref:`config_http_conn_man_headers_x-client-trace-id` headers.
+
+.. _config_http_conn_man_headers_x-ot-span-context:
+
+x-ot-span-context
```
Comment (mattklein123) [inline docs/configuration/http_conn_man/headers.rst]: Is it worth documenting that the actual format of this is base64 "binary OT carrier?" This is quite LS specific currently. I guess I would probably mention that this is LS specific and when Zipkin support is added we can update the docs to cover the differences between the 2 providers.

Comment (RomanDzhabarov) [inline docs/configuration/http_conn_man/headers.rst]: Yup, it def makes sense and it's LS specific

Comment (RomanDzhabarov) [inline docs/configuration/http_conn_man/headers.rst]: updated

**Pull request merged**